src/ConfigFactory.Core/IConfigModule.cs

```diff
@@ -48,10 +48,10 @@ public interface IConfigModule
     /// <returns>
     /// <see langword="true"/> if the validation was successful; <see langword="false"/> if the validation failed
     /// </returns>
-    public virtual bool Validate() => Validate(out _, out _);
+    public bool Validate() => Validate(out _, out _);
 
     /// <inheritdoc cref="Validate()"/>
-    public virtual bool Validate(out string? message) => Validate(out message, out _);
+    public bool Validate(out string? message) => Validate(out message, out _);
```

Comment:
No reason to hide this member

```suggestion
    public virtual bool Validate(out string? message) => Validate(out message, out _);
```

Reply:
(same reply as above - virtual keyword presence in the interface method does not make any difference, it is still virtual even without the "vritual" keyword)